subsys/bluetooth/host/gatt.c

```diff
@@ -2539,20 +2545,93 @@ int bt_gatt_notify_cb(struct bt_conn *conn,
 }
 
 #if defined(CONFIG_BT_GATT_NOTIFY_MULTIPLE)
+
+static int bt_gatt_notify_multiple_cb(struct bt_conn *conn, uint16_t num_params,
```

Comment:
This function seems to have a single place it's called from, `bt_gatt_notify_multiple()`, and it's really the only thing that the calling function does, so I'd suggest just to merge the content of `bt_gatt_notify_multiple_cb()` into `bt_gatt_notify_multiple()`, and then remove this function.